CoreDailies.cs

```diff
@@ -414,10 +421,19 @@ public void DeathKnightLord()
             return;
         Core.Logger("Daily: Death KnightLord Class");
         if (Core.CheckInventory("DeathKnight Lord", toInv: false))
+        {
+            Core.Logger("You already own DeathKnight Lord Class, Skipped");
             return;
+        }
         if (!CheckDaily(492, true, "Shadow Skull"))
-            return;
-        DailyRoutine(492, "bludrut4", "Shadow Serpent", "Shadow Scales", 5);
+         
+            DailyRoutine(492, "bludrut4", "Shadow Serpent", "Shadow Scales", 5);
+            Core.FarmingLogger("Shadow Skull", 30, "Shadow Skull");
```

Comment:
Please leave the 3rd param of FarmingLogger empty, also move farming logger above Daily Routine